mail/isync/Portfile

```diff
@@ -1,48 +1,71 @@
-description         command line utility to synchronize mailboxes
-long_description    isync is a command line utility for synchronizing IMAP4 \
-                    and Maildir mailboxes. With it one can read mail while \
-                    offline, and later synchronize new messages, message \
-                    deletions and flag changes in a fine-grained manner when \
-                    an internet connection is available.
-homepage            http://isync.sourceforge.net/
-master_sites        sourceforge:project/isync/isync/${version}/
-
-checksums           rmd160  877afe4f90961ed9fb1edef109e28a4fd6289845 \
-                    sha256  0d36dbb57bb06c8bbe10bb66f40ae543095b143443209b7037167be600420150 \
-                    size    336281
-
-compiler.c_standard 2011
-
-depends_build       port:perl5 \
-                    port:pkgconfig
-
-depends_lib         port:cyrus-sasl2 \
-                    port:db53 \
-                    path:lib/libssl.dylib:openssl \
-                    port:zlib
-
-depends_run         path:share/curl/curl-ca-bundle.crt:curl-ca-bundle
+PortSystem              1.0
+
+name                    isync
+version                 1.4.1
+revision                1
+categories              mail
+platforms               darwin
+license                 GPL-2
+maintainers             {gmail.com:emanuele.giaquinta @exg} openmaintainer
+description             command line utility to synchronize mailboxes
+long_description        isync is a command line utility for synchronizing IMAP4 \
+                        and Maildir mailboxes. With it one can read mail while \
+                        offline, and later synchronize new messages, message \
+                        deletions and flag changes in a fine-grained manner when \
+                        an internet connection is available.
+homepage                http://isync.sourceforge.net/
+master_sites            sourceforge:project/isync/isync/${version}/
+
+checksums               rmd160  877afe4f90961ed9fb1edef109e28a4fd6289845 \
+                        sha256  0d36dbb57bb06c8bbe10bb66f40ae543095b143443209b7037167be600420150 \
+                        size    336281
+
+compiler.c_standard     2011
+
+depends_build           port:perl5 \
+                        port:pkgconfig
+
+depends_lib             port:cyrus-sasl2 \
+                        port:db53 \
+                        path:lib/libssl.dylib:openssl \
+                        port:zlib
+
+depends_run             path:share/curl/curl-ca-bundle.crt:curl-ca-bundle
+
+startupitem.create      no
+startupitem.location    LaunchAgents
 
 # Patch the sample configuration to use MacPorts certificates and
 # drv_proxy_gen.pl to use MacPorts perl for building on OS X 10.8 and
 # earlier: https://sourceforge.net/p/isync/bugs/37/
 patchfiles          patch-paths.diff
 
+post-extract {
+    xinstall -m 644 -W ${filespath} isync.plist ${worksrcpath}
```

Comment:
Please add `startupitem.name mbsync` and rename the plist file to `mbsync.plist`. isync is the name of the legacy wrapper that has been removed in mbsync 1.4.

Reply:
Done